torch/onnx/_internal/fx/fx_utils.py

```diff
@@ -0,0 +1,287 @@
+"""Utilities for manipulating the torch.fx object."""
```

Comment:
I am inclined to find a more descriptive name than utils. Because once something is called utils, everything pours in. In particular, I wouldn't consider the `_ATENLIB_FUNCTIONS` a utility for onnx fx.

Reply:
Looks like the file contains the following components that can be further divided.
* ops registration and dispatching.
* decomposition.
* export option.

Reply:
Would adding function_dispatcher.py for atenlib and decomposition tables and export_options.py for ExportOptions be satisfactory? And move op validation back to exporter.py